mdoc/Test/en.expected-frameworks/FrameworksIndex/One.xml

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Framework Name="One">
+  <Type Name="MyFramework.MyNamespace.MyClass">
+    <Member Sig="public double OnlyInClassic { get; set; }" />
```

Comment:
Since you mentioned that you'd be fine with [adding the "DocId" in v5.0 for everyone](https://github.com/mono/api-doc-tools/issues/31#issuecomment-282885080), and per @dend's request ... I'm going to use this new "DocId" as the identifier for the FrameworksIndex files. 

Mentioning here in this thread to give a chance for thoughts if there are any :)